cli/tests/pcluster/config/test_validators.py

```diff
@@ -545,6 +546,69 @@ def test_s3_validator(mocker, boto3_stubber, config, num_calls, bucket, expected
     utils.assert_param_validator(mocker, config, expected_message)
 
 
+@pytest.mark.parametrize(
+    "bucket, error_code, expected_message",
+    [
+        (
+            "bucket",
+            None,
+            "cluster_resource_bucket must be in the same region of the cluster.",
+        ),
+        (
+            "not_existed_bucket",
+            "NoSuchBucket",
+            "The S3 bucket 'not_existed_bucket' does not appear to exist",
+        ),
+        (
+            "access_denied_bucket",
+            "AccessDenied",
+            "You do not have access to the S3 bucket 'access_denied_bucket'",
+        ),
+    ],
+)
+def test_s3_bucket_region_validator(mocker, boto3_stubber, error_code, bucket, expected_message):
+    os.environ["AWS_DEFAULT_REGION"] = "eu-west-1"
```

Comment:
While this is ok, would be nice if we can test out the different cases when region is provided with CLI vs. provided in original config.
If that logic is already tested, you can also leave a comment in the code to point to an existing test case here

Reply:
Done! Here we provide the region by using os.environ["AWS_DEFAULT_REGION"], it should be passed to the config, although we use it in many places, it seems that we don't have test for this, we may need to add this test later